COMPILING.md

```diff
@@ -114,6 +114,25 @@ files.
    complete set of built binaries can be found in `./build/bin` once the build
    is complete.
 
+## Compilation on Windows using Visual Studio
+
+Follow these instructions to work on Windows with Visual Studio:
```

Comment:
It would be useful to make clear whether that's on top of the above instructions, or a replacement of (some of) the steps.